pkg/ctl/enable/repo.go

```diff
@@ -88,11 +90,20 @@ func doEnableRepository(cmd *cmdutils.Cmd) error {
 		return nil
 	}
 
-	userInstructions, err := installer.Run(context.Background())
+	userInstructions, fluxSSHKey, err := installer.Run(context.Background())
 	if err != nil {
 		logger.Critical("unable to set up gitops repo: %s", err.Error())
 		return err
 	}
+
+	if fluxSSHKey != nil {
```

Comment:
Why not move this logic inside the installer instead?

Reply:
@martina-if It's just that I wanted not to bloat the scope of the installer.

If you're fine, I can definitely move this inside the installer, either in this PR or in another PR.

WDYT?

Reply:
I think for now it makes more sense inside the installer so let's move it there.

Reply:
Addressed in 2dba2c88. Please note that I had to replicate `flux.PublicKey` struct in the `deploykey` package to cut import cycle between `flux` and `deploykey` packages.